packages/manager/modules/enterprise-cloud-database/src/card/card.scss

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
Should we use `sass` or `less` ?

Reply:
I thought `sass` was the technology of choice for the Manager ?